core/common/src/main/java/alluxio/Constants.java

```diff
@@ -118,323 +118,7 @@
-  public static final String USER_LINEAGE_ENABLED = "alluxio.user.lineage.enabled";
-  public static final String USER_FILE_WAITCOMPLETED_POLL_MS =
-      "alluxio.user.file.waitcompleted.poll.ms";
-  public static final String USER_UFS_DELEGATION_ENABLED =
-      "alluxio.user.ufs.delegation.enabled";
-  public static final String USER_UFS_DELEGATION_READ_BUFFER_SIZE_BYTES =
-      "alluxio.user.ufs.delegation.read.buffer.size.bytes";
-  public static final String USER_UFS_DELEGATION_WRITE_BUFFER_SIZE_BYTES =
-      "alluxio.user.ufs.delegation.write.buffer.size.bytes";
-
-  /** alluxio-fuse related conf keys */
-
-  /**
-   * Passed to fuse-mount, maximum granularity of write operations:
-   * Capped by the kernel to 128KB max (as of Linux 3.16.0).
-   */
-  public static final String FUSE_MAXWRITE_BYTES = "alluxio.fuse.maxwrite.bytes";
-  /** Have the fuse process log every FS request. */
-  public static final String FUSE_DEBUG_ENABLE = "alluxio.fuse.debug.enabled";
-  /** Maximum number of Alluxio paths to cache for fuse conversion. */
-  public static final String FUSE_CACHED_PATHS_MAX = "alluxio.fuse.cached.paths.max";
-  public static final String FUSE_DEFAULT_MOUNTPOINT = "alluxio.fuse.mount.default";
-  public static final String FUSE_FS_ROOT = "alluxio.fuse.fs.root";
-  /** FUSE file system name. */
-  public static final String FUSE_FS_NAME = "alluxio.fuse.fs.name";
-
-  public static final String OSS_ACCESS_KEY = "fs.oss.accessKeyId";
-  public static final String OSS_SECRET_KEY = "fs.oss.accessKeySecret";
-  public static final String OSS_ENDPOINT_KEY = "fs.oss.endpoint";
-
-  public static final String S3N_ACCESS_KEY = "fs.s3n.awsAccessKeyId";
-  public static final String S3N_SECRET_KEY = "fs.s3n.awsSecretAccessKey";
-
-  // Not prefixed with fs, the s3a property names mirror the aws-sdk property names for ease of use
-  public static final String S3A_ACCESS_KEY = "aws.accessKeyId";
-  public static final String S3A_SECRET_KEY = "aws.secretKey";
-
-  public static final String GCS_ACCESS_KEY = "fs.gcs.accessKeyId";
-  public static final String GCS_SECRET_KEY = "fs.gcs.secretAccessKey";
-
-  public static final String SWIFT_USER_KEY = "fs.swift.user";
-  public static final String SWIFT_TENANT_KEY = "fs.swift.tenant";
-  public static final String SWIFT_API_KEY = "fs.swift.apikey";
-  public static final String SWIFT_PASSWORD_KEY = "fs.swift.password";
-  public static final String SWIFT_AUTH_URL_KEY = "fs.swift.auth.url";
-  public static final String SWIFT_AUTH_PORT_KEY = "fs.swift.auth.port";
-  public static final String SWIFT_AUTH_METHOD_KEY = "fs.swift.auth.method";
-  public static final String SWIFT_USE_PUBLIC_URI_KEY = "fs.swift.use.public.url";
-  public static final String SWIFT_AUTH_KEYSTONE = "keystone";
-  public static final String SWIFT_AUTH_SWIFTAUTH = "swiftauth";
-  public static final String SWIFT_SIMULATION = "fs.swift.simulation";
+  public static final String LOGGER_TYPE = PropertyKey.LOGGER_TYPE;
```

Comment:
Otherwise to obtain the logger, our code will be like
`private static final Logger LOG = LoggerFactory.getLogger(PropertyKey.LOGGER_TYPE.toString());` if we change `PropertyKey` into `enum`, which looks redundant.